### DISCUSSION TITLE
aide de construction dun site de commerce vente en lignes des ordinateurs des telephones et tot genres

### DISCUSSION THREAD

**Discussion by mohamdoul-haphase-ba** [twbs/bootstrap#40824] [Q&A]
je veux que vous m'aider a développer cela J'ai commence mais a l'intérieur j'ai des problèmes

**Comment by julien-deramond** [comment]
Hey @mohamdoul-haphase-ba  
I'm closing this discussion as it's not a precise question regarding/linked to Bootstrap.
If you need any help from the Bootstrap community, no problem. However, you'll need to ask precise questions about your issues and probably share some code parts that would allow the community to help you.
